AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+kdanmobile.com##div[class^="styled__ContentWrapper-"]
```

Comment:
This is a wide rule - it blocks some content. Before you open a new pull request, please check old rules for the website.
I've changed the current rule.